build.py

```diff
@@ -85,3 +87,27 @@ def set_properties(project):
 @depends('run_unit_tests')
 def test(project, logger):
     pass
+
+@task
+@description("Run a negative test that will fail due to exception.")
+def negative_test(project, logger):
+    print("Running a -ve test")
+    command = ExternalCommandBuilder('hydra', project)
+    command.use_argument('negative-test-exception')
+    result = command.run_on_production_source_files(logger)
```

Comment:
Yes, these are two test cases one is guaranteed to succeed, other is guaranteed to fail (By raising an exception).
Right now they don't do anything, but I wanted them to be integrated in the pipeline to test proper error reporting.